docs/resources/smartpool_settings.md

```diff
@@ -0,0 +1,150 @@
+---
+# Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Mozilla Public License Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://mozilla.org/MPL/2.0/
+#
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+title: "powerscale_smartpool_settings resource"
+linkTitle: "powerscale_smartpool_settings"
+page_title: "powerscale_smartpool_settings Resource - terraform-provider-powerscale"
+subcategory: ""
+description: |-
+  This resource is used to manage the SmartPools Settings of PowerScale Array. We can Create, Update and Delete the SmartPools Settings using this resource.Note that, SmartPools Settings is the native functionality of PowerScale. When creating the resource, we actually load SmartPools Settings from PowerScale to the resource.
+---
+
+# powerscale_smartpool_settings (Resource)
+
+This resource is used to manage the SmartPools Settings of PowerScale Array. We can Create, Update and Delete the SmartPools Settings using this resource.  
+Note that, SmartPools Settings is the native functionality of PowerScale. When creating the resource, we actually load SmartPools Settings from PowerScale to the resource.
+
+
+## Example Usage
+
+```terraform
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Available actions: Create, Update, Delete and Import
+# If resource arguments are omitted, `terraform apply` will load SmartPools Settings from PowerScale, and save to
+# terraform state file.
+# If any resource arguments are specified, `terraform apply` will try to load SmartPools Settings (if not loaded) and update the settings.
```

Comment:
Maybe it's better to clarify the meaning of Delete here as well, because it doesn't really delete anything on the array unlike other resources, just stop managing on Terraform right?

Reply:
Good idea, added.